### PR TITLE
 Implement UCS get pollers data feature

### PR DESCRIPTION
**Pull request by pengz1**
Background
UCSM are updating some information. To keep those information updated, we need to create pollers to poll  and update data in RackHD.

Details:
1. Add .gitignore document for better git practice
2. Create an API and controller for /poller API
3. Implemented Unit test for new features.

Paired with @bbcyyb

@iceiilin @anhou @mcgg

### CODE REVIEW

File: swagger/swagger.yaml [hunk 1]
```diff
@@ -133,6 +132,47 @@ paths:
           description: "Successful data fetch of rackmount catalogs"
       x-tags:
       - tag: "default"
+
+  /pollers:
```
Comment (anhou) [inline swagger/swagger.yaml]: It looks like the content is described in responses description'"Successful data fetch of catalogs by class id"' so maybe it don't need to add new /pollers, just leverage /catalog API.  I thinks there is no poller here, just it's named by RackHD's /poller. so there's no direct relationship here

Comment (pengz1) [reply swagger/swagger.yaml]: /pollers and /catalog will get different data from UCSM.
Most data of /pollers are device status, like cpuStats while /catalog is usually device static information.

Comment (anhou) [reply swagger/swagger.yaml]: Thanks for clarification

**Comment by PengTian0** [comment]
test this please

**Comment by bbcyyb** [comment]
+1

**Comment by iceiilin** [comment]
test this please

**Pull request merged**